updates/testing.json

```diff
@@ -75,16 +75,6 @@
           "start_percentage": 1.0
         }
       }
-    },
-    {
-      "version": "35.20220103.2.0",
-      "metadata": {
-        "rollout": {
-          "duration_minutes": 2880,
-          "start_epoch": 1641423600,
-          "start_percentage": 0.0
-        }
-      }
```

Comment:
I assume this is safe to do in Cincinnati? Is `35.20220103.2.0` no longer going to be in the update graph at all. What do existing nodes on `35.20220103.2.0` do when this happens?

Reply:
The node exists in the graph as long as it's in the build list, but it won't have any inbound or outbound edges, same as if we had released but not started a rollout yet.  So they'll just sit where they are.

When we push a new version, the node should get an outbound edge same as the other releases.  We should manually verify though.

Reply:
It will be in the graph, it just won't be connected.
Graph nodes come from [the release index](https://builds.coreos.fedoraproject.org/prod/streams/testing/releases.json), which we're not reverting here.